### PR TITLE
I think this is good rn

### PR DESCRIPTION
**Pull request by m00nwtchr**
(also please make a 1.6.1-hunter tag after you merge this)

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -1,6 +0,0 @@
-[submodule "lib/websocketpp"]
```
Comment (ruslo) [inline .gitmodules:1]: Please remove all changes that are not related to hunterization:
- https://docs.hunter.sh/en/latest/creating-new/patch.html

```
Please do push commits only related to hunterization
```

File: CMakeLists.txt [hunk 2]
```diff
@@ -1,84 +1,113 @@
...
 target_compile_definitions(sioclient_tls PRIVATE -DSIO_TLS)
-if(BUILD_SHARED_LIBS)
-set_target_properties(sioclient_tls
-	PROPERTIES
-		SOVERSION ${MAJOR}
-		VERSION ${MAJOR}.${MINOR}.${PATCH}
-	)
-endif()
+if (BUILD_SHARED_LIBS)
+    set_target_properties(sioclient_tls
```
Comment (ruslo) [inline CMakeLists.txt:67]: If it's not critical, no need to touch it. Try to make minimal changes.

**Review by ruslo — CHANGES_REQUESTED** [review]

File: .gitmodules (hunk 1 above)
Comment (m00nwtchr) [reply .gitmodules:1]: well, this is sorta related cause the submodules are no longer required so no point including them?

Comment (ruslo) [reply .gitmodules:1]: No need for hard remove, just don't use it in CMakeLists.txt. It's a source of conflict for future versions.

**Comment by ruslo** [comment]
> madmongo1 merged commit 574e682 into hunter-packages:hunter-1.6.1 

@madmongo1 Have you seen "ruslo requested changes 20 hours ago" ?
